Gruntfile.js

```diff
@@ -201,6 +201,8 @@ module.exports = function (grunt) {
         src: ['<%= yeoman.app %>/index.html'],
         ignorePath:  /\.\.\//,
         exclude: [
+          // bower registration error? we get 2x versions of uri.js/urijs
```

Comment:
- excluding the dup of `uri.js`
- excluding the `uri.js` mods we dont need, rather than overriding `main` in `bower.json`